liubov.tarasenko/Gemfile

```diff
@@ -1 +1,6 @@
+# frozen_string_literal: true
+source 'https://rubygems.org'
```

Comment:
please fix rubocop

Reply:
done. should I resolve situation when url isn`t finish with extension?

Reply:
check my comment. there is a better solution. The image does not always have .png in the URL. check `headers content type`